static/homepage.js

```diff
@@ -29,10 +61,18 @@
       y: {
         label: "Votes Cast",
         min: 0,
-        max: 4000000,
```

Comment:
dropping this line does prevent us from seeing the "high water mark", but the upside is that it's a lot easier to see the detail on the D vs R graph when it's zoomed in.

Reply:
(i now fixed this by leaving the max in place when not viewing the partisan split)